### PR TITLE
Fix browserify performance

### PR DESCRIPTION
**Pull request by christophwitzko**
Hi @gr2m,

I added some speed improvements for the `/store.js` route and replace uuid.js with a similar module.

Have a nice day,
Christoph

### CODE REVIEW

File: lib/routes/client.js [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
-    })
-    b.require(path.resolve(__dirname, '../../client'))
-    b.bundle(function (error, buffer) {
-      if (error) throw error
-
-      reply(buffer)
-        .type('application/javascript')
-    })
+    if (STOREJS) return reply(STOREJS).type('application/javascript')
+    reply(Boom.badRequest('store.js is not yet available'))
```
Comment (gr2m) [inline lib/routes/client.js]: Can we avoid the `badRequest` response? Maybe we can use a Promise for that, and once it resolves, it returns the bundled JS code? Then in future requests the Promise is already resolved and the Server will respond immediately?

Comment (christophwitzko) [inline lib/routes/client.js]: Thank you for the input. I fixed it. :+1:

**Comment by gr2m** [comment]
beautiful :ok_hand:

**Comment by christophwitzko** [comment]
Thank you! :+1: :relaxed:

**Comment by christophwitzko** [comment]
I just realised that `browserify` already emits events :speak_no_evil: 
That means this could be simplified a lot. I will tackle this tomorrow. :+1:

**Comment by gr2m** [comment]
ha! Great catch!

**Comment by christophwitzko** [comment]
Sadly the `browserify` events do not work in this case. :/
We can leave everything as it is.

**Pull request merged**